pandas/core/base.py

```diff
@@ -951,22 +957,36 @@ def max(self):
         >>> idx.max()
         ('b', 2)
         """
+        nv.validate_minmax_axis(axis)
         return nanops.nanmax(self.values)
 
-    def argmax(self, axis=None):
+    def argmax(self, axis=None, skipna=True):
         """
         Return a ndarray of the maximum argument indexer.
 
+        Parameters
+        ----------
+        axis : {None}
+            Dummy argument for consistency with Series
+        skipna : bool, default True
+
         See Also
         --------
         numpy.ndarray.argmax
         """
+        nv.validate_minmax_axis(axis)
         return nanops.nanargmax(self.values)
```

Comment:
Doesn't this hit https://github.com/pandas-dev/pandas/issues/24265?

Reply:
Not this line, but the relevant line in indexes.datetimelike does.  With the commit just pushed, this fixed 24265